helm/memcached-operator/helm-charts/memcached/values.yaml

```diff
@@ -1,108 +1,34 @@
-## Memcached image and tag
-## ref: https://hub.docker.com/r/library/memcached/tags/
-##
-image: memcached:1.5.12-alpine
-
-## Specify a imagePullPolicy
-## 'Always' if imageTag is 'latest', else set to 'IfNotPresent'
-## ref: http://kubernetes.io/docs/user-guide/images/#pre-pulling-images
-##
-# imagePullPolicy:
-#
-
-## Replica count
-replicaCount: 3
-
-## Pod disruption budget minAvailable count
-## Ensure this value is lower than replicaCount in order to allow a worker
-## node to drain successfully
-pdbMinAvailable: 2
-
-## Select AntiAffinity as either hard or soft, default is hard
-AntiAffinity: "soft"
-
+AntiAffinity: hard
+affinity: {}
+extraContainers: ""
+extraVolumes: ""
+image: memcached:1.5.20
+kind: StatefulSet
 memcached:
-  ## Various values that get set as command-line flags.
```

Comment:
Note that the specs are the same. And then, we should keep the comments. 
Also, the AntiAffinity needs to be soft and we can upgrade the values for the images as follows.

> `image: memcached:1.5.20`
> `image: quay.io/prometheus/memcached-exporter:v0.6.0`
>